### PR TITLE
Document return type of `add_to_queue`

### PR DESCRIPTION
**Pull request by pmatos**

### CODE REVIEW

File: soco/core.py [hunk 1]
```diff
@@ -1369,6 +1369,9 @@ def add_to_queue(self, queueable_item, position=0, as_next=False):
                 added. Default is 0 (add URI at the end of the queue).
             as_next (bool): Whether this URI should be played as the next
                 track in shuffle mode. This only works if `play_mode=SHUFFLE`.
+
```
Comment (ghcs27) [inline soco/core.py]: The blank line contains a whitespace, which flake8 doesn't like. Could you please remove it?

File: soco/core.py [hunk 2]
```diff
@@ -1369,6 +1369,9 @@ def add_to_queue(self, queueable_item, position=0, as_next=False):
                 added. Default is 0 (add URI at the end of the queue).
             as_next (bool): Whether this URI should be played as the next
                 track in shuffle mode. This only works if `play_mode=SHUFFLE`.
+                
+        Returns:
+            An integer representing the index of the new item in the queue.
```
Comment (ghcs27) [inline soco/core.py]: To comply with the Napoleon docstring style, this should be
`int: The index of the new item in the queue.`

File: soco/core.py (hunk 1 above)
Comment (pmatos) [inline soco/core.py]: You're right. I am using the github editor and didn't notice that. Sorry.

**Comment by pmatos** [comment]
I have now reviewed this and it should be ok to merge?

**Comment by ghcs27** [comment]
I'd say everything is alright.

**Comment by ghcs27** [comment]
This was incorporated into #540. Thank you for the contribution!

**Pull request closed without merging**